src/main/java/core/basesyntax/impl/ReadingCsvFileImpl.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.impl;
+
+import core.basesyntax.service.CantWorkWithThisFileException;
+import core.basesyntax.service.ReadingCsvFile;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class ReadingCsvFileImpl implements ReadingCsvFile {
+    @Override
+    public List<String> reading(String fileName) {
+        try {
```

Comment:
Consider using try-with-resources to ensure that all resources are closed properly. In this case, Files.readAllLines() does not require it as it closes resources internally, but keep this in mind for future reference when dealing with I/O operations.